changedetectionio/tests/test_diff_filter_changes_as_add_delete.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python3
+
```

Comment:
```suggestion
+ @NOTE:  THIS RELIES ON SOME MIDDLEWARE TO MAKE CHECKBOXES WORK WITH WTFORMS UNDER TEST CONDITION, see changedetectionio/tests/util.py
```

good to add a little pointer which may save someone a lot of time in the future